src/encoded/cart_view.py

```diff
@@ -1,14 +1,26 @@
 from pyramid.view import view_config
 from collections import OrderedDict
+from snovault import COLLECTIONS
+from pyramid.httpexceptions import HTTPBadRequest
 
+CART_USER_MAX = 30  # Maximum number of non-deleted carts allowed per user
 
 def includeme(config):
     # config.scan()
-    # config.add_route('carts', '/carts{slash:/?}')
     config.add_route('cart-view', '/cart-view{slash:/?}')
+    config.add_route('cart-manager', '/cart-manager{slash:/?}')
     config.scan(__name__)
 
 
+def get_cart_objects_by_user(request, userid):
```

Comment:
We shouldn't override this function with status and permission filtering to encourage reuse later. (In other words since the function is called `get_cart_objects_by_user` it should just get the cart objects by user.)

Reply:
Changed, and added a separate function for filtering.